unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/TaskbarHUD/TaskbarHUDController.cs

```diff
@@ -133,15 +147,59 @@ private void ChatHeadsGroup_OnHeadOpen(TaskbarButton taskbarBtn)
         OpenPrivateChatWindow(head.profile.userId);
     }
 
+    private void View_OnSettingsToggleOn()
+    {
+        settingsHud.SetVisibility(true);
+        OnAnyTaskbarButtonClicked?.Invoke();
+    }
+
+    private void View_OnSettingsToggleOff()
+    {
+        settingsHud.SetVisibility(false);
+    }
+
+    private void View_OnBackpackToggleOn()
+    {
+        avatarEditorHud.SetVisibility(true);
+        OnAnyTaskbarButtonClicked?.Invoke();
+    }
+
+    private void View_OnBackpackToggleOff()
+    {
+        avatarEditorHud.SetVisibility(false);
+    }
+
+    private void View_OnExploreToggleOn()
+    {
+        exploreHud.SetVisibility(true);
+        OnAnyTaskbarButtonClicked?.Invoke();
+    }
```

Comment:
It was a requirement for the explore popup to be able to have it on screen while chatting

Reply:
I have added a piece of code to take on account this specific rule. It's working. Thanks Pato!